src/Search.js

```diff
@@ -13,11 +22,13 @@ const Search = () => {
             <div className="search-row">
               <input
                 type="text"
+                value={searchInput}
                 id="customerName"
                 className="form-control"
+                onChange={handleSearchInput}
                 placeholder="Customer name"
               />
-              <button className="btn btn-primary">Search</button>
+              <SearchButton />
             </div>
           </form>
```

Comment:
I think the search functionality is not working. You just need to define a `handleSumbit` in the parent component of `Search` component and implement `filter` functionality. Don't forget to use `prevent.default()` and then pass your `handleSubmit` to `form` tag
Would be good to do some study about [submitting form in React](https://www.w3schools.com/react/react_forms.asp#:~:text=Adding%20Forms%20in%20React&text=This%20will%20work%20as%20normal,let%20React%20control%20the%20form.)

Reply:
done